src/catalog/database_catalog.cpp

```diff
@@ -1753,7 +1753,7 @@ common::ManagedPointer<execution::functions::FunctionContext> DatabaseCatalog::G
 common::ManagedPointer<execution::functions::FunctionContext> DatabaseCatalog::GetFunctionContext(
     const common::ManagedPointer<transaction::TransactionContext> txn, catalog::proc_oid_t proc_oid) {
   auto func_ctx = GetProcCtxPtr(txn, proc_oid);
-  NOISEPAGE_ASSERT(!(func_ctx == nullptr && IS_BUILTIN_PROC(proc_oid)),
+  NOISEPAGE_ASSERT(!(func_ctx == nullptr && pg_proc_.IsBuiltinProc(proc_oid)),
                    "Builtin procedures should have been bootstrapped.");
```

Comment:
This is the only use of `IS_BUILTIN_PROC`/`pg_proc_.IsBuiltinProc` which makes me think that it is actually unnecessary and possibly shouldn't even be tracked explicitly since it should be implicitly tracked by the functions existing in the `pg_catalog` namespace.

Reply:
Sounds good.